node/cn/tracers/api.go

```diff
@@ -567,6 +579,13 @@ func (api *API) StandardTraceBadBlockToFile(ctx context.Context, hash common.Has
 // executes all the transactions contained within. The return value will be one item
 // per transaction, dependent on the requestd tracer.
 func (api *API) traceBlock(ctx context.Context, block *types.Block, config *TraceConfig) ([]*txTraceResult, error) {
+	if !api.unsafeTrace {
+		if atomic.LoadInt64(&heavyAPIRequestCount) > int64(HeavyAPIRequestLimit) {
```

Comment:
How about to remove type casting int64 by redeclaring the type to int64. i.e., `var HeavyAPIRequestLimit int64 = 0 `(Same as L790)

Reply:
Right. I have redeclared the type. The type is int32 since I changed the atomic load function from int64 to int32.
```
var (
	HeavyAPIRequestLimit int32 = 500
	heavyAPIRequestCount int32 = 0
)
```